navigation/ca_move_base/launch/move_base_flex.launch

```diff
@@ -0,0 +1,57 @@
+<launch>
+  <arg name="robot_id"  default="$(optenv ID 1)"      doc="Unique identifier of the robot [1-Inf.)"/>
+  <arg name="ns"        value="create$(arg robot_id)" doc="Namespace of the robot. By default: create1"/>
+  <arg name="tf_prefix" value="$(arg ns)_tf"          doc="Tf prefix"/>
+
+  <arg name="laser" default="$(optenv LASER rplidar)" doc="Laser for mapping the environment"/>
+
+  <arg name="global_planner" value="$(optenv GLOBAL_PLANNER gbp)"/>
+  <arg name="local_planner"  value="$(optenv LOCAL_PLANNER teb)"/>
+
+  <node pkg="mbf_costmap_nav" type="mbf_costmap_nav" respawn="false" name="move_base_flex" output="screen">
+
+    <!-- Load move base params, since mbf requires the same ones -->
+    <rosparam file="$(find ca_move_base)/config/move_base_params.yaml" command="load" />
+
+    <rosparam file="$(find ca_move_base)/config/costmap_common_params.yaml" command="load" ns="global_costmap" />
+    <rosparam file="$(find ca_move_base)/config/global_costmap_params.yaml" command="load" />
+
+    <rosparam file="$(find ca_move_base)/config/$(arg global_planner)_global_planner_params.yaml" command="load" />
+
+    <rosparam file="$(find ca_move_base)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
+    <rosparam file="$(find ca_move_base)/config/local_costmap_params.yaml" command="load" />
+
+    <rosparam file="$(find ca_move_base)/config/$(arg local_planner)_local_planner_params.yaml" command="load" />
+
+    <!-- We want a single map for multi-robots -->
+    <remap from="map" to="/map"/>
+    <remap from="/laser/scan" to="/$(arg ns)/$(arg laser)/scan"/>
+    <remap from="/mobile_base/sensors/bumper_pointcloud"
+           to="mobile_base/sensors/bumper_pointcloud"/>
+
+    <remap from="odom" to="/$(arg ns)/odom"/>
+    <remap from="cmd_vel" to="/$(arg ns)/cmd_vel"/>
```

Comment:
Removing the namespaces from the topic names work well. Notice that I removed the starting `/` too.

```suggestion
    <remap from="/laser/scan" to="$(arg laser)/scan"/>
    <remap from="/mobile_base/sensors/bumper_pointcloud"
           to="mobile_base/sensors/bumper_pointcloud"/>

    <remap from="odom" to="odom"/>
    <remap from="cmd_vel" to="cmd_vel"/>
```

Reply:
Can you address this in `move_base.launch` too?

Reply:
@eborghi10 qq, aren't the changes in line 32 and 33 the same as skipping the remapping?

Reply:
IDK if you saw my comment above: 

> Notice that I removed the starting / too.

You don't need `ns` in this case because the node leaves inside the namespace `ns` already so it will be appended automatically if you don't add a slash at the beginning.

Reply:
Done!